app/BUILD.bazel

```diff
@@ -178,7 +178,7 @@ VIEW_MODELS = [
     "src/main/java/org/oppia/android/app/hintsandsolution/SolutionViewModel.kt",
     "src/main/java/org/oppia/android/app/home/HomeItemViewModel.kt",
     "src/main/java/org/oppia/android/app/home/promotedlist/ComingSoonTopicListViewModel.kt",
-    "src/main/java/org/oppia/android/app/home/recentlyplayed/OngoingStoryViewModel.kt",
+    "src/main/java/org/oppia/android/app/home/recentlyplayed/PromotedStoryViewModel.kt",
```

Comment:
Path for `PromotedStoryViewModel` need to be updated

Previously it was 
https://github.com/oppia/oppia-android/blob/f26d769fc232d7a21b1c3406daa7198a678a2224/app/BUILD.bazel#L138

Reply:
I'm sorry this was out of confusion. this can be resolved now